backend/src/main/kotlin/com/ksidelta/libruch/modules/organisation/UserOrganisationsReadModel.kt

```diff
@@ -11,16 +13,21 @@ import javax.persistence.EmbeddedId
 import javax.persistence.Entity
 
 @Service
-class UserToOrganisationsEventProcessor(val repository: UserToOrganisationsModelRepository) {
+class UserToOrganisationsEventProcessor(
+    val repository: UserToOrganisationsModelRepository,
+    val eventGateway: EventGateway
+) {
 
-    @EventHandler
+    @EventHandler(payloadType = MemberAdded::class)
```

Comment:
I hate that I need to add a type here if I am using DomainEventMessage...
Hasn't spring solved it somehow?